gamemodes/terrortown/gamemode/client/cl_help/populate_appearance.lua

```diff
@@ -0,0 +1,554 @@
+local materialIcon = Material("vgui/ttt/vskin/helpscreen/appearance")
+
+local function PopulateGeneralPanel(parent)
+	local form = CreateForm(parent, "header_global_color")
+
+	form:MakeHelp({
+		label = "help_color_desc"
+	})
+
+	-- store the reference to the checkbox in a variable
+	-- because the other settings are enabled based on
+	-- the state of this checkbox
+	local enbColor = form:MakeCheckBox({
+		label = "label_global_color_enable",
+		initial = GLAPP.ShouldUseGlobalFocusColor(),
+		onChange = function(_, value)
+			GLAPP.SetUseGlobalFocusColor(value)
+		end,
+		default = false
+	})
+
+	form:MakeColorMixer({
+		label = "label_global_color",
+		initial = GLAPP.GetFocusColor(),
+		onChange = function(_, color)
+			GLAPP.SetFocusColor(color)
+		end,
+		master = enbColor
+	})
+
+	form:MakeHelp({
+		label = "help_scale_factor"
+	})
+
+	form:MakeSlider({
+		label = "label_global_scale_factor",
+		min = 0.1,
+		max = 3,
+		decimal = 1,
+		initial = GLAPP.GetGlobalScale(),
+		onChange = function(_, value)
+			GLAPP.SetGlobalScale(value)
+		end,
+		default = GLAPP.GetDefaultGlobalScale()
+	})
+end
+
+local function PopulateHUDSwitcherPanelSettings(parent, currentHUD)
+	parent:Clear()
+
+	parent:MakeHelp({
+		label = "help_hud_special_settings"
+	})
+
+	for key, data in pairs(currentHUD:GetSavingKeys() or {}) do
+		if data.typ == "color" then
+			parent:MakeColorMixer({
+				label = data.desc or key,
+				initial = currentHUD[key],
+				onChange = function(_, color)
+					currentHUD[key] = color
+
+					if isfunction(data.OnChange) then
+						data.OnChange(currentHUD, color)
+					end
+				end,
+				showAlphaBar = true,
+				showPalette = true
+			})
+
+
+		elseif data.typ == "number" then
+			parent:MakeSlider({
+				label = data.desc or key,
+				min = 0.1,
+				max = 4,
+				decimal = 1,
```

Comment:
I think we should rethink the data behind the SavingKeys. The values min = 0.1, max = 4 and decimal = 1 were only for the scale, but might be useless for some other number value. But since this should be tackled in the HUD code, it would be better to remove the whole ```elseif data.typ == "number" then" part, since it is not used in our current HUDs.

Reply:
but it CAN be used. Each HUD can add their own saving keys. Actually i probably should add a boolean as well.
But yes, `min`, `max` and `decimal` should be defined by the saving keys

Reply:
So then it should be either tackled now or be removed in the mean time, since for me it is quite confusing, why min, max and decimal are hardcoded.

Reply:
I can change this, yes. But honestyl I don't want to do it in this PR. But if  you want to have it now, I can implement it today

Reply:
done